cockatrice/src/messagelogwidget.cpp

```diff
@@ -677,13 +677,15 @@ void MessageLogWidget::logSetCardCounter(Player *player, QString cardName, int c
 
 void MessageLogWidget::logSetCounter(Player *player, QString counterName, int value, int oldValue)
 {
-    if (counterName == "life") {
+    QString counterDisplayName = QString(counterName);
```

Comment:
There’s no reason to add the display string here. If the “if statement” works, then construct the display name. You’re excessively creating strings (expensive) that may never get used.